message/legacy/replace.go

```diff
@@ -74,27 +73,33 @@ func quoteString(buf *bytes.Buffer, s string) {
 	}
 }
 
+var utf16enc = unicode.UTF16(unicode.LittleEndian, unicode.IgnoreBOM).NewEncoder()
+
 // InternalV8Binary does some funky v8 magic
 // new Buffer(in, "binary") returns soemthing like (u16 && 0xff)
 func InternalV8Binary(in []byte) ([]byte, error) {
-	var u16 bytes.Buffer
-	enc := unicode.UTF16(unicode.LittleEndian, unicode.IgnoreBOM).NewEncoder()
-	trans := transform.NewWriter(&u16, enc)
-	if _, err := io.Copy(trans, bytes.NewReader(in)); err != nil {
-		return nil, fmt.Errorf("internalV8bin: failed to transform input to u16: %w", err)
+	guessedLength := len(in) * 2
```

Comment:
If someone knows the exact formula please let me know. I think it is `1.5` going off the top of my head but I may be completely wrong. It is fine to go overboard here as it won't cause any real performance problems (we just care that we are doing fewer allocs not how large they are) but I want to make sure that we are not erroring out on weird input.